### DISCUSSION TITLE
How to keep launcher open after ED exits?

### DISCUSSION THREAD

**Discussion by InnerBushman** [rfvgyhn/min-ed-launcher#111] [General]
I want the launcher to stay open so i can quickly switch between versions. Now it quits every time ED quits. I do not use any command line switches. Please advise?

**Comment by rfvgyhn** [comment]
That's not something that's currently supported. The closest thing would be creating multiple shortcuts with command line switches pointing to different versions.

**Reply by InnerBushman** [reply]
It might sound like a silly question but what's the /AUTOQUIT do then?

**Reply by rfvgyhn** [reply]
It closes the launcher after the game shuts down. If you don't use it, the launcher will stay open to allow reading the output.

When the launcher was first written, the flags were meant to be exact duplicates of the default launcher so that you wouldn't need to change them when going between the default launcher and this one.

**Reply by InnerBushman** [reply]
Oh, you mean like waits for a button press before closing the terminal?
What would it take to implement going back to main menu of the mini launcher after the game quits? Right now I'm using a script to restart the launcher but that takes a lot of time. For the curious I play as a Fuel Rat and need to switch versions often and quickly to adjust my game to client's version. It would mean a world to me to not have to restart the whole launcher and wait precious seconds for all the things in the background to establish connections, check updates, etc.
We both know we wouldn't be here if the original launcher wasn't so damn slow and buggy (freezes when I select legacy).
Please let me know if this functionality is something that's doable without enormous amounts of effort. I would really appreciate it.

**Reply by rfvgyhn** [reply]
> Oh, you mean like waits for a button press before closing the terminal?

Yeah. Doesn't make much sense for a terminal app, but that's the case.

It shouldn't be a big deal to loop back to the menu. It won't work via Epic though for the [same reason](https://github.com/rfvgyhn/min-ed-launcher#min-launcher-specific) the `/restart` flag doesn't work.

**Reply by InnerBushman** [reply]
That's good to hear. I play only on STEAM so there should be any troubles here. Would you like me to add a feature request in the issues tab?
